include/rocksdb/db.h

```diff
@@ -808,24 +808,36 @@ class DB {
   // stats should be included, or file stats approximation or both
   enum SizeApproximationFlags : uint8_t {
     NONE = 0,
-    INCLUDE_MEMTABLES = 1,
+    INCLUDE_MEMTABLES = 1 << 0,
     INCLUDE_FILES = 1 << 1
   };
 
+  struct SizeApproximationOptions {
+    // Defines whether the returned size should include the recently written
+    // data in the mem-tables (if the mem-table type supports it), data
+    // serialized to disk, or both.
+    SizeApproximationFlags include_flags = INCLUDE_FILES;
+  };
```

Comment:
We put all those options in file options.h. Maybe we should find a good home to all those in the future, but right now I suggest we keep it there.

Reply:
ok